src/v/config/throughput_control_group.cc

```diff
@@ -240,7 +243,16 @@ Node convert<security::acl_principal>::encode(const type& principal) {
         break;
     }
     case security::principal_type::ephemeral_user: {
-        // not supported yet, invalid config produced if it appears
+        // TBD: how to define the PP/SR ephemeral principal in one place
+        // and avoid dependency of `config` on `pandaproxy`?
+        YAML::Node service_node = node[ids::service];
+        if (principal.name() == "__pandaproxy") {
```

Comment:
@BenPope I'm looking for an advice from you what would be a good way to share this literal between `pandaproxy` and `config`.

Reply:
I guess we can move them to `src/v/config`.